app/grandchallenge/archives/views.py

```diff
@@ -558,6 +564,115 @@ def get_serializer_class(self):
             return ArchiveItemSerializer
 
 
+class ArchiveItemJobListView(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    JobsList,
```

Comment:
What are you inheriting this for? The only thing that is common between the JobList and this archive item job list is the column definition, right? Everything else is overwritten. I think it would be better to inherit from `PaginatedTableListView` directly.